tensorflow_probability/python/bijectors/real_nvp.py

```diff
@@ -206,24 +209,37 @@ def _bijector_fn(x0, input_depth, **condition_kwargs):
         validate_args=validate_args,
         name=name)
 
+  @property
+  def num_masked(self):
```

Comment:
I don't think we need for this to be public, would you mind making this private instead? I'm particularly concerned that this will raise an error if `_input_depth` is `None`. We could document this, but if we don't need to have this public, then there's no need to worry about this.

Could make it a `_get_num_masked()` to avoid the name collision.